### PR TITLE
[RW-8461] Bump base image and all sub images to fix leo_url in cromwell python script

### PR DESCRIPTION
**Pull request by yonghaoy**
The change that I need: https://github.com/broadinstitute/cromwhelm/pull/34

### CODE REVIEW

File: terra-jupyter-aou/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 2.1.6 - 2022-06-23T10:58:12.961300Z
+
+- Update `terra-jupyter-base` to `1.0.10`
+  - Fix leo_url variable in workspace_cromwell.py ****script for AoU projects
```
Comment (calbach) [inline terra-jupyter-aou/CHANGELOG.md:4]: what's with the `****` ?

**Review by calbach — APPROVED** [review]
FYI: as for why this change does anything, it's because this line pulls from master: https://github.com/DataBiosphere/terra-docker/blob/c45d2d22075990ebbdd7373606bf8dc47de650a5/terra-jupyter-base/Dockerfile#L142

**Comment by Qi77Qi** [comment]
can we use a permlink for the script? that way we'll know exactly what version of the script we're pulling

**Comment by calbach** [comment]
From offline debugging with @yonghaoy , it seems likely that at least some of these failures (e.g. out of disk) are caused by conflicts with existing versions from this reverted PR. @Qi77Qi I think we should flag any version created in this PR as "do not use" in respective changelogs, if there are any others that Yonghao is not addressing in the PR here.

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**